python/tank/commands/core_upgrade.py

```diff
@@ -181,6 +182,20 @@ def _run(self, log, core_version):
             log.error(msg)
             return_status = {"status": "update_blocked", "reason": msg}
 
+        elif status == TankCoreUpdater.UPDATE_BLOCKED_BY_CONFIG:
```

Comment:
This is not really required for the fix, but I thought I would add it in, since we shouldn't try and update any configs that are immutable.